labelbox/schema/bulk_import_request.py

```diff
@@ -274,3 +277,26 @@ def create_from_local_file(cls,
             response_data = _send_create_file_command(client, request_data,
                                                       file_name, file_data)
         return cls(client, response_data["createBulkImportRequest"])
+
+
+class NdjsonError(Exception):
+    pass
+
+
+class UuidError(NdjsonError):
+    pass
```

Comment:
I'd be careful here - I guess we will add more validation to the SDK, so how about we make a base class more generic? And inherit from LabelboxError?

Reply:
ah good call, updated